index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="toolbar-container">
+      <a href=""
+        ><img class="logo" src="./images/header/My Logo.png" alt="logo"
+      /></a>
+      <nav>
+        <a href=""
+          ><img
+            class="nav-logo"
+            src="./images/header/Hamburger_icon.svg.png"
+            alt="logo"
+        /></a>
+      </nav>
+    </header>
+    <main class="headline">
+      <section>
+        <div class="name"><h1>I’m Ali Hassan. Glad to see you!</h1></div>
+        <div class="introduction">
+          <p>
+            I'm a software developer! I can help you build a product, feature or
+            website. Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </p>
+        </div>
+        <div class="connect">
+          <p>LET'S CONNECT</p>
+        </div>
+        <div>
+          <ul class="social">
+            <li>
+              <a href="#"
+                ><img src="./images/main/Vector-3.png" alt="twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/main/Linkedin icon.svg" alt="linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/main/M.svg" alt="M" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/main/Lomri.svg" alt="github" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/main/V.svg" alt="V" /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
```

Comment:
- [ OPTIONAL ] Lastly, at the bottom of your design, I noticed that you're missing the curved border bottom effect as seen in the Figma design. Your design needs to look exactly like the design so kindly implement this in your design as well :wink: 

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://github.com/anshuman7negi/my-portfolio/assets/102699958/f6d1d26b-d19e-46b2-b569-31d18a620fec) | ![image](https://github.com/anshuman7negi/my-portfolio/assets/102699958/ac35333a-82f2-4569-995e-ed38e5fcf027) |

Reply:
@NahnahAJ Please help me both are not working 
border-bottom-left-radius: 100px;
border-radius: 0 0 100px 0;

Reply:
- [ ] I noticed that you asking for help with the border-radius. You have a good guidance from @shella12  [right here](https://github.com/mhassa204/Portfolio-Project/pull/1#discussion_r1219038713). Please effect these changes and test the project. You can also contact me on zoom.
 - [ ] Please also look our for overriding CSS rules that may prevent your styles from taking effect. You can use **!important** to test if the rule is being overridden. NOTE: the use of important is discouraged but its a good tool for checking if a rule is not applying because it overridden. You should delete it after and write your rules to cascade correctly. 👍🏾